active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,150 @@
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+## Create a Breadcrumb
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+ ~~~~ 
+     <muxc: Breadcrumb ItemsSource = {"Node", "Node", "Node"} /> 
+~~~~
+More examples to be added
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | provides the content of the Breadcrumb | Null |
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  | N/A |
+
+Note that V2 of Breadcrumb will add flyouts from chevrons to view children of a node. Additional properties will be added to the API to enable this functionality. 
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+## API
+~~~~
+
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Object Item { get; };
+}
+
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+unsealed runtimeclass BreadcrumbLayout : NonVirtualizingLayout
+{
+    BreadcrumbLayout();
+}
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
+{
+    Breadcrumb();
+
+    Object DropdownItemTemplate{ get; set; };
```

Comment:
"Drop down" is two words, so this should capitalize the second 'd': `DropDownItemTemplate`.

Reply:
Is "Drop down" treated as two words elsewhere? Drop-down is the adjective used here, which is [either hyphenated or one word](https://www.merriam-webster.com/dictionary/drop-down).

Reply:
The DropDownButton control is also written with "Down" capitalized so it would make sense to do that here too. That capitilzation is also used in multiple different APIs and lightweight styling resource names.

Reply:
Telerik also uses [DropDown]( https://docs.telerik.com/devtools/wpf/controls/radmulticolumncombobox/dropdown-properties#height-of-the-dropdown) so for consistency I'd say we could go with DropDown, but there are other cases in css where the properties are capitalized as Dropdown in the titles but the property is written as dropdown in code, so that would seem inconclusive

Reply:
Good point about it being a word. The difficulty with language is that it evolves :). The ToolTip API name bugs me, but when it was first created maybe it wasn't a word yet.

"Dropdown" makes sense, but we have a [DropDownButton](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.DropDownButton) today and [ComboBox.DropDownClosed](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ComboBox.DropDownClosed), so I think we should stick with local precedence.

Reply:
See Alberto's PR including updates to the API here https://github.com/microsoft/microsoft-ui-xaml-specs/pull/111/files